model_compression_toolkit/exporter/model_exporter/keras/fakely_quant_keras_exporter.py

```diff
@@ -72,49 +72,43 @@ def _unwrap_quantize_wrapper(layer: Layer):
             assert self.is_layer_exportable_fn(layer), f'Layer {layer.name} is not exportable.'
```

Comment:
this assert is now meaningless because you have checks in is_layer_exportable_fn